### PR TITLE
feat (header): Improve the header information

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**
SD-2742

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

File: client/app/scripts/superdesk/directives/sdSlider.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+         * @scope {Object} list - list of options
+         * @scope {Boolen} disabled - disable or enable slider functionality
+         * @scope {Object} invert - inverts min and max value
+         * @scope {Function} update - callback when slider value is changed
+         *
+         */
+        directive('sdSlider', function () {
+            return {
+                transclude: true,
+                replace: true,
```
Comment (petrjasek) [inline client/app/scripts/superdesk/directives/sdSlider.js]: afaik replace is deprecated

Comment (pavlovicnemanja92) [inline client/app/scripts/superdesk/directives/sdSlider.js]: Is there some alternative for this? Currently this is working, and also I saw that we use this on multiple places.

Comment (petrjasek) [inline client/app/scripts/superdesk/directives/sdSlider.js]: no, but is there any need for this? it will be removed in 2.0

Comment (pavlovicnemanja92) [inline client/app/scripts/superdesk/directives/sdSlider.js]: :+1: Small refactoring and this works without `replace`! :)

**Pull request merged**